src/dashboard/src/main/migrations/0013_upload_archivesspace_inherit_notes.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+def data_migration(apps, schema_editor):
+    arguments = ' '.join([
+        '--host "%host%"',
+        '--port "%port%"',
+        '--user "%user%"',
+        '--passwd "%passwd%"',
+        '--dip_location "%SIPDirectory%"',
+        '--dip_name "%SIPName%"',
+        '--dip_uuid "%SIPUUID%"',
+        '--restrictions "%restrictions%"',
+        '--object_type "%object_type%"',
+        '--xlink_actuate "%xlink_actuate%"',
+        '--xlink_show "%xlink_show%"',
+        '--use_statement "%use_statement%"',
+        '--uri_prefix "%uri_prefix%"',
+        '--access_conditions "%access_conditions%"',
+        '--use_conditions "%use_conditions%"',
+        '--inherit_notes="%inherit_notes%"',
+    ])
+
+    StandardTaskConfig = apps.get_model('main', 'StandardTaskConfig')
+    StandardTaskConfig.objects.filter(pk='10a0f352-aeb7-4c13-8e9e-e81bda9bca29').update(arguments=arguments)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0012_file_id_text'),
```

Comment:
It might make sense to add a dependency on the ArchivesSpace change `('administration', '0004_archivesspaceconfig_inherit_notes.py'),`  Would need to be tested, since I haven't tried multiple dependencies before.